src/app/config/config.yml

```diff
@@ -26,6 +26,7 @@ framework:
         name: %zikula.session.name%
         handler_id: session.handler.legacy
         storage_id: session.storage.legacy
+        cookie_httponly: false
```

Comment:
I don't understand. you are setting it to false, but saying that it doesn't work if set to false?